### DISCUSSION TITLE
Set BC version for local PTE development

### DISCUSSION THREAD

**Discussion by JlT74** [microsoft/AL-Go#230] [Q&A] [answered]
I might have missed something obvious, but is there a way to set wich BC docker image gets installed when running the localDevEnv.ps1 script if we don't want the latest version downloaded?

/Johan

**Comment by freddydk** [comment, marked as answer]
In the .AL-Go\settings.json file you can specify the artifacts used, using the setting artifact. Documented here:
https://github.com/microsoft/AL-Go/blob/main/Scenarios/settings.md#advanced-settings
if you set artifact to 
```
bcartifacts/sandbox/20.5.12345.12345//closest
```
You will use the closest artifacts to that specific version for the country specified in the country setting.
Since bcartifacts and sandbox are default, this would be the same as:
```
//20.5.12345.12345//closest
```
and if you want a specific version, you can use:
```
//21.0.46256.48157
```
or you can also specify an artifactUrl like:
```
https://bcartifacts.azureedge.net/sandbox/21.0.46256.48157/us
```
